src/js/Controllers/UIController.js

```diff
@@ -203,6 +203,18 @@ class UIController {
                 }
                 this.listener.send(RpcFactory.SuccessResponse(rpc));
                 return null
+            case "UnsubscribeButton":
+                    store.dispatch(subscribeButton(
```

Comment:
@LiliiaShlikhtLuxoft Why is there in `UnsubscribeButton` request `subscribeButton` is used? Is it similar requests? Seems the logic for `SubscribeButton` and for `UnsubscribeButton` is similar. Take a look code from 193-205 and 206-217. Seems it is copy pasted code with no changes

Reply:
@ValeriiMalkov I did it, because subscribeButton contains entity "isSubscribed", which I should change in Unsubscribe to "false", it makes buttons disappeared.

Reply:
@LiliiaShlikhtLuxoft If the code is similar you can write it like:
```
case "SubscribeButton":
case "UnsubscribeButton":
                    store.dispatch(subscribeButton(
                        rpc.params.appID,
                        rpc.params.buttonName,
                        false
                    ));
                    if(!rpc.params.buttonName || !rpc.params.appID) {
                        this.listener.send(RpcFactory.ErrorResponse(rpc, GENERIC_ERROR, "No button provide to unsubscribe"));
                        return;
                    };
                    this.listener.send(RpcFactory.SuccessResponse(rpc));
                    return null
  ```

Reply:
@ValeriiMalkov , thank you for advice! But I need "true" fro case "SubscribeButton" case and "false" for "UnsubscribeButton". I didn't get how you advice to do it. Comments in ErrorResponse are different as well.